core/controllers/base_test.py

```diff
@@ -352,6 +424,16 @@ def test_logout_page(self):
             datetime.datetime.utcnow() > datetime.datetime.strptime(
                 expiry_date[1], '%a, %d %b %Y %H:%M:%S GMT',))
 
+    def test_logout_page_with_dev_mode_disabled(self):
+        with self.swap(constants, 'DEV_MODE', False):
+            response = self.get_html_response(
+                '/logout', expected_status_int=302)
+            expiry_date = response.headers['Set-Cookie'].rsplit('=', 1)
+
+            self.assertTrue(
```

Comment:
Can you add some comments explaining what the significance of this test is?

Reply:
Actually I forgot to remove these lines, sorry about that! These lines essentially tested the expiry of cookies after logging out. But this test is irrelevant here as it is only testing the logout page with dev mode disabled